src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -134,4 +135,29 @@ private boolean personsAreUnique(List<Person> persons) {
         }
         return true;
     }
+
+    /**
+     * Sorts the list of persons in ascending order of their names.
+     */
+    public void sortList() {
+        internalList.sort(new SortAlphabetically());
+    }
+
+    /**
+     * SortAlphabetically implements a comparator class, to sort the list of persons in alphabetical order.
+     */
+    class SortAlphabetically implements Comparator<Person> {
+
+        /**
+         * Sorts in ascending order of the names of persons.
+         *
+         * @param a the first person.
+         * @param b the second person.
+         * @return an int value after comparing the names of the two persons.
+         */
+        @Override
+        public int compare(Person a, Person b) {
+            return a.getName().fullName.compareTo(b.getName().fullName);
```

Comment:
It might be more clear if variables are extracted from this line.

```suggestion
            String personAFullName = a.getName().fullName;
            String personBFullName = b.getName().fullName;
            return personAFullName.compareTo(personBFullName);
```